### PR TITLE
Update Japanese documents to follow original changes from ecda198 to 3574667

### PR DESCRIPTION
**Pull request by munierujp**
[x] Documentation update
[ ] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

Update Japanese documents to follow original changes from ecda198 to 3574667

**Which issue (if any) does this pull request address?**

none

**Is there anything you'd like reviewers to focus on?**

- Markdown syntax
- Common codes and scripts which does not depend on human  language

### CODE REVIEW

File: docs/README-ja.md [hunk 1]
```diff
@@ -57,18 +57,18 @@
   - [でもこれは本当のウェブ標準ではありません！](#but-this-isnt-a-real-web-standard)
   - [自動フォーマッターはありますか？](#is-there-an-automatic-formatter)
   - [ファイルを無視するには？](#how-do-i-ignore-files)
-  - [特定の警告を非表示にするには？](#how-do-i-hide-a-certain-warning)
+  - [ルールを無効にするには？](#how-do-i-disable-a-rule)
```
Comment (munierujp) [inline docs/README-ja.md:60]: Internal link works correctly.

File: docs/README-ja.md [hunk 2]
```diff
@@ -57,18 +57,18 @@
...
   - [私はグローバル名前空間を汚染するライブラリを使用しています。"variable is not defined"というエラーを防ぐには？](#i-use-a-library-that-pollutes-the-global-namespace-how-do-i-prevent-variable-is-not-defined-errors)
   - [実験的なJavaScriptの機能（ES Next）を使用するには？](#how-do-i-use-experimental-javascript-es-next-features)
   - [FlowやTypeScriptのようなJavaScriptの代替言語を使用できますか？](#can-i-use-a-javascript-language-variant-like-flow-or-typescript)
   - [Mocha、Jest、Jasmine、QUnitなどはどうすれば？](#what-about-mocha-jest-jasmine-qunit-etc)
   - [Web WorkersとService Workersはどうすれば？](#what-about-web-workers-and-service-workers)
+  - [警告とエラーの違いは？](#what-is-the-difference-between-warnings-and-errors)
```
Comment (munierujp) [inline docs/README-ja.md:66]: Internal link works correctly.

File: docs/README-ja.md [hunk 3]
```diff
@@ -554,6 +521,16 @@ Service workersには、かわりに次のコメントを追加してくださ
 /* eslint-env serviceworker */
 ```
 
+<h2 id="what-is-the-difference-between-warnings-and-errors">警告とエラーの違いは？</h2>
+
+`standard`は、すべてのルール違反をエラーとして扱います。つまり、0以外の終了コード（エラー）で終了するということです。
+
+しかしながら、我々はときどき`standard`のユーザーの大多数に影響を与えうるルールを変更するような、新しいメジャーバージョンをリリースすることがあります（たとえば、`var`から`let`、`const`への移行など）。これを行なうのは、利点がコストに見合うと考えられ、かつルールが[自動修正可能](#is-there-an-automatic-formatter)な場合に限られます。
+
+このような状況では、ルールの変更を「警告」に留めた「移行期間」を設けています。警告は、`standard`に0以外の終了コード（エラー）に返させません。しかしながら、警告メッセージは依然としてコンソールに表示されます。移行期間中に`standard --fix`を使うと、次のメジャーバージョンに備えてあなたのコードが更新されます。
```
Comment (munierujp) [inline docs/README-ja.md:530]: Fix typo

```suggestion
このような状況では、ルールの変更を「警告」に留めた「移行期間」を設けています。警告は、`standard`に0以外の終了コード（エラー）を返させません。しかしながら、警告メッセージは依然としてコンソールに表示されます。移行期間中に`standard --fix`を使うと、次のメジャーバージョンに備えてあなたのコードが更新されます。
```

**Review by munierujp — COMMENTED** [review]
I did a review myself.

File: docs/README-ja.md [hunk 4]
```diff
@@ -554,6 +521,16 @@ Service workersには、かわりに次のコメントを追加してくださ
...
 
+<h2 id="what-is-the-difference-between-warnings-and-errors">警告とエラーの違いは？</h2>
+
+`standard`は、すべてのルール違反をエラーとして扱います。つまり、0以外の終了コード（エラー）で終了するということです。
+
+しかしながら、我々はときどき`standard`のユーザーの大多数に影響を与えうるルールを変更するような、新しいメジャーバージョンをリリースすることがあります（たとえば、`var`から`let`、`const`への移行など）。これを行なうのは、利点がコストに見合うと考えられ、かつルールが[自動修正可能](#is-there-an-automatic-formatter)な場合に限られます。
+
+このような状況では、ルールの変更を「警告」に留めた「移行期間」を設けています。警告は、`standard`に0以外の終了コード（エラー）を返させません。しかしながら、警告メッセージは依然としてコンソールに表示されます。移行期間中に`standard --fix`を使うと、次のメジャーバージョンに備えてあなたのコードが更新されます。
+
+我々は、`standard`でゆっくりと慎重なアプローチに励んでいます。我々は一般的に、新しい言語機能の使用を強制することに関して極めて保守的です。我々は`standard`を気軽で楽しいものにしたいので、あなたの妨げになるような変更には気をつけています。いつも通り、必要に応じていつでもルールを無効にすることができます。
```
Comment (munierujp) [inline docs/README-ja.md:532]: Improve Japanese

```suggestion
我々は、`standard`でゆっくりと慎重なアプローチに励んでいます。我々は一般的に、新しい言語機能の使用を強制することに関して極めて保守的です。我々は`standard`を気軽で楽しいものにしたいので、その妨げになるような変更には留意しています。いつも通り、必要に応じていつでもルールを無効にすることができます。
```

**Review by feross — APPROVED** [review]

**Comment by feross** [comment]
Thanks @munierujp – we're all super grateful for your work to keep the Japanese translation up-to-date!

**Pull request merged**